File: Data Structures & Algorithms/remove-element/submission-0.java
class Solution {
    
    public int removeElement(int[] nums, int val) {
        int c=0,i=0,j=nums.length-1;
        while(i<=j){
           if(nums[i]!=val){
              i++;
              continue;
           }
           if(nums[j]==val){
            j--;
            continue;
           }
           int h=nums[i];
           nums[i]=nums[j];
           nums[j]=h;
        i++;
        j--;
           }
        
        return i;
    }
}

File: Data Structures & Algorithms/max-consecutive-ones/submission-0.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        cnt=0
        c=0
        n=len(nums)
        for i in nums:
            if i==1:
                c+=1
                cnt=max(cnt,c)
            else:
                c=0
        return cnt
            